docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -168,6 +182,7 @@ $channel_id   = get_term_meta( $channel->term_id, \BigCommerce\Taxonomies\Channe
 ```
 
 The channel ID is not available anywhere on the client side.
+
 ### Customers
 
 A logged out user does not have any customer information. For a logged in user, you can create a Customer object to get the customer's information.

```

Comment:
A logged out user does not have any customer information. For a logged-in user, you can create a Customer object to get the customer's information.